src/index.tsx

```diff
@@ -82,11 +82,11 @@ const initializeCourseViewer = (): void => {
 
   let prevScrollpos = window.scrollY;
   window.onscroll = () => {
-    displayDesktopScrollToTop();
-    handleMobileNavOnScroll();
+    handleDesktopScrollToTopDisplay();
+    handleMobileNavigationOnScroll();
   };
 
-  const displayDesktopScrollToTop = (): void => {
+  const handleDesktopScrollToTopDisplay = (): void => {
     if (!topButton)
```

Comment:
Please add the curly braces here. This is javascript not python lol

Reply:
I tend to disagree, it was supposed to be a one-liner except prettier decided to split it

Reply:
But since it's not a one liner any more, just add the braces back.

Reply:
FINE prettier does this for a bunch of other things too

Reply:
Umm...The braces still aren't there lol